itest/e2e_test.go

```diff
@@ -51,7 +51,7 @@ func TestCovenantEmulatorLifeCycle(t *testing.T) {
 }
 
 func TestQueryPendingDelegations(t *testing.T) {
```

Comment:
Shouldn't we add e2e test for this case i.e one message fails , one not, and one of them is delivered. ?

Reply:
It was a bit difficult to write a test that the message would fail in e2e, so what I did to test was very dumb, comment the send in batches and only use the send multiple txs 

If this is a must, I will investigate a way to add an e2e test